src/utils/version-compatible.ts

```diff
@@ -0,0 +1,17 @@
+import {Context} from '../context';
+import {ApiVersion} from '../base_types';
+
+/**
+ * Check if the current or optionally supplied version is compatible with a given version
+ */
+export default function versionCompatible(referenceVersion: ApiVersion, currentVersion: ApiVersion = Context.API_VERSION): boolean {
```

Comment:
This function will be super helpful in the future, thanks for adding it!

Reply:
Great!